### ISSUE TITLE
Explanation video for the RChain community blog

### ISSUE THREAD

**Issue by Valentine-Mario** [rchain/bounties#957]
Benefit to RChain

This is an effort to enhance the marketing of the <a href="https://rchaincommunity.xyz">RChain community blog</a>. It is a 2 part video, the first explaining the importance of the forum to the community while the second explains all the functions of the blog and how to use it.

_Estimated Budget of Task:_ **$2700**
_Estimated Timeline Required to Complete the Task:_ **[2 weeks]**
_How will we measure completion?_ **[Creation of the video content]**

**Comment by Viraculous** [comment]
@Valentine-Mario, I would suggest that you suspend this work for the meantime because there have been some contentions around this RWWC.xyz unitll things are fully fledged out. BTW the way has this gone through the TAC?

**Comment by Valentine-Mario** [comment]
One of the main aim of the forum this month is to increase publicity. And that is what this issue helps to solve. Hopefully this issue would aid other RAM appreciate the forum

**Comment by nonnykul** [comment]
@**Viraculous**, this project is to rejuvenate the importance of forum towards building a Community.
We have to appreciate what we have, since its the only official RChain forum.

**Comment by Cryptovideos** [comment]
Yes - since iv seen the steps and script i agree completely - much value from this, appreciate the guide/steps shown. Clean and to the point -will go a long way to assist new Rams/ people like me who have never taken the opportunity to appreciate and utilize a forum like this.

**Comment by owans** [comment]
This video will go a long way in simplifying user experience,and also aid in creating awareness of the importance of the blog to members.

**Comment by jplepel** [comment]
If we are looking for more outward facing exposure, why don't we push the use of Reddit more. It can work as a forum and there is already a user-base.

It's not that co-op members don't know how to use the forum, its that they don't want to, or see the need of it. I will not be supporting this bounty

**Comment by luigidemeo** [comment]
Why do we need a tutorial for a blog? I agree that we should be posting on reddit threads, why have our own interface when there is one that works?

**Comment by Cryptovideos** [comment]
@jplepel great feedback - what would you suggest exactly..**why don't we push the use of Reddit more** what do you think would be more effective?

**Comment by Viraculous** [comment]
Firstly @Valentine-Mario, @nonnykul, I am not saying you should publicize the forum but making a simple request to suspend publicity around this for the meant time "until" things are fully fledged out. Secondly, since the creation if the RChain official website and blog, I haven't really seen any marketing around them on the bounty but my question now is where should we best direct our marketing efforts ?  RWC.xyz or the official RChain website and blog?, Thirdly, like I initially asked, has the gone through TAC?

**Comment by Cryptovideos** [comment]
@luigidemeo thanks for dropping in ** I agree that we should be posting on reddit threads** lets post video links and such? people much prefer watching and learning via video module dont you think? any suggestion that would be better suited, would be appreciated?  how do we approach this better?

**Comment by Cryptovideos** [comment]
@Viraculous **but my question now is where should we best direct our marketing efforts ? ** very good question....what do you think the right answer is?

**Comment by jplepel** [comment]
@Cryptovideos I think a collective effort to participate on reddit would be great. There doesn't have to be a bounty or how-to video behind this effort either. Out everyone's pure interest in RChain, they should be more that happy to upvote and comment on posts. @decentralizebanksy is currently the only one posting consistently.  

Any content that will be or has been posted in the .xyz will get more exposure on reddit. We can even go further outside of /r/RChain/ and post on other related subreddits.

**Comment by luigidemeo** [comment]
agree with @jplepel . We need to be thought leaders, writing posts on rchains official reddit as well as other reddits that relate to rchain explaining why it is a valuable alternative to other chains. I do not think blogging about rchain on rchains official blog should be paid. People write about ethereum all day long with no payment because they want to be solidity thought leaders.

**Comment by Cryptovideos** [comment]
@jplepel @luigidemeo and everyone else? whats your vibe? comment here and lets either Hold off and do this another time , reduce the budget to cover our time and move along? but i dont want to upset the apple cart or create something if there's gonna be division or negativity between us on this...genuinely thought this was a swell idea....but i hear you as well Lui and Jpl
